### PR TITLE
MIJN-4637 - Voeg marge toe indien geen lees meer knop 

### PR DESCRIPTION
**Pull request by janthijs**

### CODE REVIEW

**Comment by timvanoostrom** [comment]
Maybe it's better to add the margin for Dashboard specific. There is a scss selector for that: 

```
@include select-component(MyNotifications) {
  @include mq-mobile() {
    margin-bottom: 3rem;
  }
}
```

**Comment by timvanoostrom** [comment]
Have you considered that?

**Comment by janthijs** [comment]
> Maybe it's better to add the margin for Dashboard specific. There is a scss selector for that:
> 
> ```
> @include select-component(MyNotifications) {
>   @include mq-mobile() {
>     margin-bottom: 3rem;
>   }
> }
> ```

Goede suggestie, heb het toegepast.

File: src/client/pages/Dashboard/Dashboard.module.scss [hunk 1]
```diff
@@ -54,15 +54,19 @@
     border-top: 1px solid $color-neutral-grey2;
     margin: 0 1rem;
 
-    @include media-q('>phone') {
+    @include mq-phone() {
+      margin-bottom: 3rem !important;
```
Comment (timvanoostrom) [inline src/client/pages/Dashboard/Dashboard.module.scss:58]: Is deze `!important` vereist? Misschien is 't beter om de specificity te tunen

**Review by timvanoostrom — APPROVED** [review]

**Pull request merged**